applications/StructuralMechanicsApplication/tests/test_patch_test_large_strain.py

```diff
@@ -77,6 +77,35 @@ def _define_movement(self,dim):
             b[2] = 0.7
         
         return A,b
+    
+    def _define_movement_UL(self,dim):
+        if(dim == 2):
```

Comment:
this displacements are very small. Why don't you call here the function of the TL?